### ISSUE TITLE
Misnamed homework branch

### ISSUE THREAD

**Issue by raxonpc** [coders-school/cpp-fundamentals#18]
nwd-nnw should be named nwd-nww

**Comment by ziobron** [comment]
🏅 2 XP granted for @raxonpc
Thanks for raising this Issue to help to make the course better!

**Comment by ziobron** [comment]
That's true. This caused other Issue with not granting points for this task. The easier solution was to rename the task name to the not proper version. Sorry, this issue will be fully fixed after the course. Right now this is a workaround not to confuse people who are not familiar yet with git.
